crates/pallet-domains/src/staking.rs

```diff
@@ -1655,17 +1644,14 @@ pub(crate) mod tests {
                 operator_free_balance - operator_total_stake - ExistentialDeposit::get()
             );
 
-            // cannot register with same operator key
+            // registering with same operator key is allowed
```

Comment:
Is this what we want, or is there another way we should check for duplicate operator registrations?

Reply:
This should be okay. Operators are free to choose their signing key and is no incentive to reuse the signing key albeit operators are at risk using same signing key. But that would not hurt the protocol since all operators need to stake individually and all compromised operators will be slashed accordingly if they used same signing key and key is compromised.

Will also spend sometime thinking on this further but first pass looks good to me

Reply:
The signing key is a public key, the operator is free to use the same signing key as others, but without the private key it won't be able to produce a bundle, if it has the private key that means:
- The signing key is compromised, the best we can do at the protocol side is to provide a way to change the operator signing key
- Or someone is intentionally reusing the same signing key for registering multiple operators, this doesn't seem to hurt anything, and the user needs to specify `--operator-id` to indicate which operator the node is running for